src/headless/converse-chatboxes.js

```diff
@@ -709,6 +727,16 @@ converse.plugins.add('converse-chatboxes', {
                     to_jid = stanza.getAttribute('to');
                 }
 
+                const requests_receipt = !_.isNull(stanza.querySelector('request'));
```

Comment:
Any reason why you're not incuding the xmlns in this query?

Like you do above: `const receipt = sizzle(`received[xmlns="${Strophe.NS.RECEIPTS}"]`, stanza).pop(); `

Reply:
Not really :-) Added it.